insights/specs/datasources/compliance/__init__.py

```diff
@@ -227,7 +227,7 @@ def policy_link(self, policy_id, opt):
             logger.info("Operation completed successfully.\n")
```

Comment:
can we also change `Operation completed successfully` to something more specific like `Policy successfully assigned`?